types/0_audio.ts

```diff
@@ -1,14 +1,23 @@
 import * as types from "../tl/2_types.ts";
 import { constructThumbnail, Thumbnail } from "./0_thumbnail.ts";
 
+/** This object represents an audio file to be treated as music by the Telegram clients. */
 export interface Audio {
+  /** Identifier for this file, which can be used to download or reuse the file */
   fileId: string;
+  /** Unique identifier for this file, which is supposed to be the same over time and for different bots. Can't be used to download or reuse the file. */
   fileUniqueId: string;
+  /** Duration of the audio in seconds as defined by sender */
   duration: number;
+  /** Performer of the audio as defined by sender or by audio tags */
   performer?: string;
+  /** Title of the audio as defined by sender or by audio tags */
   title?: string;
+  /** MIME type of the file as defined by sender */
```

Comment:
In case if you missed it by mistake, there is also a fileName property in Audio type, according to Bot API. Not sure if that's the case in MTProto.